exe/server/main.go

```diff
@@ -7,15 +7,17 @@ import (
 	"path/filepath"
 	"syscall"
 
+	_ "net/http/pprof"
+
 	"contrib.go.opencensus.io/exporter/prometheus"
 	logging "github.com/ipfs/go-log"
 	"github.com/textileio/fil-tools/api/server"
 	"github.com/textileio/fil-tools/tests"
-	_ "net/http/pprof"
 )
 
 var (
-	grpcHostAddr = "127.0.0.1:5002"
+	grpcHostAddr     = "0.0.0.0:5002"
+	grpcWebProxyAddr = "0.0.0.0:6002"
```

Comment:
Updated to `0.0.0.0` so we can accept connection over any network interface.

Reply:
Also @jsign when this gets merged and deployed to azure, can you open up port 6002?

Reply:
Sure, will do.